R/plumber.R

```diff
@@ -250,24 +250,34 @@ Plumber <- R6Class(
       }
 
       # Set and restore the wd to make it appear that the proc is running local to the file's definition.
+      old_wd <- NULL
       if (!is.null(private$filename)) {
         old_wd <- setwd(dirname(private$filename))
-        on.exit({setwd(old_wd)}, add = TRUE)
```

Comment:
Not sure what you're doing here with the `on.exit` changes, but presuming it has to do with the order of the cleanup happening backwards? Have you considered `on.exit(..., add = TRUE, after = FALSE)`?

Reply:
I didn't know about `after=`!